src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function NavBar({
+  firstHalfTimeCode,
+  secondHalfTimeCode,
+  interviewTimeCode,
+  countSpecials,
+}) {
+  return (
+    <NavWrapper>
```

Comment:
Do you need this wrapper? I think you could just style your ul since it is the outside element of your component.

Reply:
Yes I need this wrapper. It`s a semantic decision. with the wrapper I'm using a nav element for the list. without the wrapper it's only a list with links.